<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IvBox</class>
 <widget class="QWidget" name="IvBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DVs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="finalStats">
     <property name="title">
      <string>Final Stats</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hpivdesc">
        <property name="text">
         <string>HP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="hpivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="hpivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="attivdesc">
        <property name="text">
         <string>Att:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="attivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="attivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="defivdesc">
        <property name="text">
         <string>Def:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="defivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="defivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="spattivdesc">
        <property name="text">
         <string>Sp. Att:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="spattivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spattivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="spdefivdesc">
        <property name="text">
         <string>Sp. Def:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="spdefivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spdefivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="speedivdesc">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="speedivlabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="speedivspin">
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="abilityTitle">
       <property name="text">
        <string>&amp;Ability</string>
       </property>
       <property name="buddy">
        <cstring>ability1</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ability1">
       <property name="text">
        <string>Ability 1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ability2">
       <property name="text">
        <string>Ability 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ability3">
       <property name="text">
        <string>DW Ability</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="hiddenPowerTitle">
       <property name="text">
        <string>&amp;Hidden Power</string>
       </property>
       <property name="buddy">
        <cstring>hiddenPowerType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="hiddenPowerType"/>
     </item>
     <item>
      <widget class="QLabel" name="hiddenPowerPower">
       <property name="text">
        <string>Power: 70</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>113</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hpivspin</tabstop>
  <tabstop>attivspin</tabstop>
  <tabstop>defivspin</tabstop>
  <tabstop>spattivspin</tabstop>
  <tabstop>spdefivspin</tabstop>
  <tabstop>speedivspin</tabstop>
  <tabstop>hiddenPowerType</tabstop>
  <tabstop>ability1</tabstop>
  <tabstop>ability2</tabstop>
  <tabstop>ability3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
